django/core/checks/model_checks.py

```diff
@@ -8,12 +9,15 @@
 
 @register(Tags.models)
 def check_all_models(app_configs=None, **kwargs):
+    db_table_attr_map = defaultdict(list)
```

Comment:
Not a big fan of the variable name. What's the rationale behind the `attr` suffix? I feel like `db_table_models` or something similar would make more sense.